js/external_svg.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @file
+ * ECL fix for SVGs in IE 11.
+ */
+(function (Drupal) {
+  Drupal.behaviors.svg4Everybody = {
+    attach: function attach() {
```

Comment:
Missed this last time: no need to name the function.
```suggestion
    attach: function () {
```